docs/snapcraft-syntax.md

```diff
@@ -20,24 +20,41 @@ contain.
  * `config` (string)  
    Path to the runnable in snap that will be used as the [Snappy config
    interface](https://developer.ubuntu.com/snappy/guides/config-command/).
- * `services` (yaml subsection)  
-   A set of keys representing service names with values as defined by the
-   [Snappy packaging spec](https://developer.ubuntu.com/snappy/guides/packaging-format-apps/).
- * `binaries` (yaml subsection)  
-   A set of keys representing binary names with values as defined by the
-   [Snappy packaging spec](https://developer.ubuntu.com/snappy/guides/packaging-format-apps/).
+ * `apps` (yaml subsection)  
+   A map of keys for applications. These are either daemons or command line
+   accessible binaries.
+   * `command` (string)
+     Specifies the internal command to expose. If it is a `daemon` this
+     command is used to start the service.
+   * `daemon` (string)
+     If present, integrates the runnable as a system service. Valid values are
+     `forking` and `simple`.
+     If set to `simple`, it is expected that the command configured is the main
+     process.
+     If set to `forking`, it is expected that the configured command will call
+     fork() as part of its start-up. The parent process is expected to exit
+     when start-up is complete and all communication channels are set up.
+     The child continues to run as the main daemon process. This is the
+     behavior of traditional UNIX daemons.
+   * `stop-command` (string)
+     Requires `daemon` to be specified and represents the command to run to
+     stop the service.
+   * `stop-timeout` (integer)
+     Requires `daemon` to be specified. It is number in seconds that the
+     system will wait for the service to stop.
```

Comment:
I don't know how to change reality :-)

> Configures the time to wait for stop. If a service is asked to stop, but does not terminate in the specified time, it will be terminated forcibly via SIGTERM, and after another timeout of equal duration with SIGKILL (see KillMode= in systemd.kill(5)). Takes a unit-less value in seconds, or a time span value such as "5min 20s". Pass "0" to disable the timeout logic. Defaults to DefaultTimeoutStopSec= from the manager configuration file (see systemd-system.conf(5)).

Unless we disable the possibility of allowing 0